config.mk

```diff
@@ -32,14 +32,18 @@ CFLAGS_OPT := \
   -flto \
   -fstrict-aliasing
 
+# enable everything and then selectively disable some warnings
 CFLAGS_WARN := \
   -Weverything \
   -Werror \
-  -Wno-unknown-warning-option \
-  -Wno-reserved-id-macro \
+  -pedantic \
+  \
   -Wno-dollar-in-identifier-extension \
   -Wno-gnu-statement-expression \
-  -pedantic
+  -Wno-padded \
+  -Wno-reserved-id-macro \
+  -Wno-unknown-warning-option \
+  -Wno-unused-macros
```

Comment:
I was too considering to remove the packed pragmas and add `-Wno-packed`. seems superfluous to me.

@ijc25 any strong view on this? If not I'll update the PR with another patch